src/CONST.ts

```diff
@@ -2058,6 +2058,7 @@ const CONST = {
             INVOICE: 'invoice',
             SUBMIT: 'submit',
             TRACK: 'track',
+            GLOBAL_CREATE: 'create',
```

Comment:
NAB, instead of `GLOBAL_CREATE` maybe we should just use `CREATE`. When I think of global create, I think of the menu itself